### ISSUE TITLE
No download link found.

### ISSUE THREAD

**Issue by EoD** [MacTee/KSP-Mod-Admin-aOS#62]
Windows/Linux v2.0.0.11

Since recently I get with the very same setup as previously the following error "No download link found!"

```
[...]
[11.04.2015 04:24:29.259] [9] Info    : Update check for mod "MKS" via "GitHub" started...
[11.04.2015 04:24:29.495] [9] Info    : The mod "MKS" is outdated.
[11.04.2015 04:24:29.499] [9] Info    : Update check for mod "TextureReplacer" via "KerbalStuff" started...
[11.04.2015 04:24:29.781] [9] Info    : Error during update check for mod "TextureReplacer". Error: "Unexpected character encountered while parsing value: S. Path '', line 0, position 0."
[11.04.2015 04:24:29.784] [9] Info    : Update check for mod "Contract Window +" via "KerbalStuff" started...
[11.04.2015 04:24:30.073] [9] Info    : Error during update check for mod "Contract Window +". Error: "Unexpected character encountered while parsing value: S. Path '', line 0, position 0."
[11.04.2015 04:24:30.078] [9] Info    : Update check for mod "RemoteTech" via "GitHub" started...
[11.04.2015 04:24:30.298] [9] Info    : The mod "RemoteTech" is outdated.
[11.04.2015 04:24:30.300] [9] Info    : Downloading mod update "Ferram-Aerospace-Research"...
[11.04.2015 04:24:33.279] [9] Debug   : No download link found!
[11.04.2015 04:24:33.281] [9] Info    : Downloading mod update "DistantObject"...
[11.04.2015 04:24:34.315] [9] Debug   : No download link found!
[11.04.2015 04:24:34.317] [9] Info    : Downloading mod update "KerbalAlarmClock"...
[11.04.2015 04:24:35.432] [9] Debug   : No download link found!
[11.04.2015 04:24:35.435] [9] Info    : Downloading mod update "CrowdSourcedScience"...
[11.04.2015 04:24:36.414] [9] Debug   : No download link found!
[11.04.2015 04:24:36.418] [9] Info    : Downloading mod update "KSPTrajectories"...
[11.04.2015 04:24:37.368] [9] Debug   : No download link found!
[11.04.2015 04:24:37.370] [9] Info    : Downloading mod update "MKS"...
[11.04.2015 04:24:38.934] [9] Debug   : No download link found!
[11.04.2015 04:24:38.937] [9] Info    : Downloading mod update "RemoteTech"...
[11.04.2015 04:24:39.918] [9] Debug   : No download link found!
```

**Comment by Gribbleshnibit8** [comment]
Github changed the layout of the page, and the code relied on a very specific page layout and component class and div structure. Any fix I can do right now would be a bandaid on the fact that they could do the same again later. We'll need to make use of the github API to do it correctly, which takes more time than I have, and Mac is busy too right now. We are aware and it'll get fixed Soon(tm) hopefully.

**Comment by EoD** [comment]
Isn't a "bandaid" (workaround) a better temporary solution than rendering KSPMA non-functional?

**Comment by Gribbleshnibit8** [comment]
Probably. My first test on a fix didn't get it, but I'll try a couple more things. It didn't rank high priority for me to look at since I've been too busy to play, and only about 5 of my mods use github. I wouldn't say the program is 'non-functional' since Curse and KerbalStuff still work fine.

**Comment by EoD** [comment]
No worries, I didn't mean to demand an update from you. I just wanted to point out the problem :)

I haven't checked in detail but it seemed that KerbalStuff isn't working for me either (it's also in the error log). I thought it's related to the github errors on first sight.

**Comment by Gribbleshnibit8** [comment]
Alright, submitted pull request #63 that fixed this problem for me. I still want this to be converted to use the github API, but @MacTee has expressed concern about using octokit due to the .NET 4.5 requirement and the cross platform nature of this project.

I've spent hours looking up any alternatives as well as attempting to implement simple use of the API and I just don't know what I'm doing well enough to do it, so this is the 'bandaid' to fix it until a permanent solution can be implemented. :)

**Comment by MacTee** [comment]
fixed for now ... =/

**Comment by EoD** [comment]
Where do I get the new binary? The link still provides me with a 2.0.0.11

**Comment by MacTee** [comment]
After i have fixed the issue #42 i'll release a new version, till then you have to build it by yourself =/
